build.gradle

```diff
@@ -96,6 +96,8 @@ dependencies {
     implementation libraries.objenesis
 
     testImplementation libraries.assertj
+    testImplementation libraries.junitJupiterApi
+    testImplementation libraries.junitJupiterParams
```

Comment:
added this in as it appears junit-jupiter-params is not on the classpath at the moment. I _hope_ this is the right place to add this in as I couldn't see anywhere else that set these dependencies for the main module.

If it is wrong though, or if we'd prefer to not add another dependency, I am happy to separate the cases into individual tests instead/use `@TestFactory`